src/osd/OSD.cc

```diff
@@ -1939,6 +1939,8 @@ bool OSD::asok_command(string command, cmdmap_t& cmdmap, string format,
     store->get_db_statistics(f);
   } else if (command == "dump_scrubs") {
     service.dumps_scrub(f);
+  } else if (command == "dump_objectstore_db_histogram") {
```

Comment:
It might be a good idea to rename this something like "calculate_objectstore_db_histogram" to better suggest/convey that this is a very slow operation?